src/main/java/org/elasticsearch/action/update/UpdateRequest.java

```diff
@@ -205,105 +202,171 @@ int shardId() {
         return this.shardId;
     }
 
-    public String script() {
+    public Script scriptObject() {
```

Comment:
Now that we target this change to 2.0 only, I would just break bw compatibility, remove the old script() and call this one script()?